text/0028-indexing-csv.md

```diff
@@ -166,6 +168,7 @@ curl \
 - 🔴 Sending a payload excessing the limit will lead to a 413 Payload Too Large - **payload_too_large** error code.
 - 🔴 Sending an invalid CSV format will lead to a 400 bad_request - **malformed_payload** error code.
 - 🔴 Sending a CSV header that does not conform to the specification will lead to a 400 bad_request - **malformed_payload** error code.
+- 🔴 Sending a CSV cell with the type `number` or `boolean` that can't be parsed will lead to a 400 bad_request - **malformed_payload** error code.
```

Comment:
Is it a catch-up to the spec or a newly introduced behavior?

Reply:
The `number` part is catch-up, and the `boolean` part is introduced by this spec